src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -111,6 +111,52 @@
         [expand-button expanded? chat-id message-id])])
    {:justify-timestamp? true}])
 
+(defview contact-request-message
+  [{:keys [chat-id message-id content from timestamp-str group-chat outgoing current-public-key expanded?] :as message}]
+  (letsubs [{:keys [last-updated
+                    pending?]} [:contacts/contact-by-identity chat-id]]
+    (let [sent-by-us? (= current-public-key from)
+          pending-state? (if sent-by-us?
+                           ;; last-updated is only updated if we have been added to their contacts
+                           (not (pos? last-updated))
+                           pending?)]
```

Comment:
could it be anything but true here? How could it not be in a pending state if the contact-request-message is not from us?

Reply:
ok, first case: 

if it's sent by us, we check whether the user has replied to it (`last-updated`) -> we show it as accepted, if we don't have a `last-updated` -> still pending

second case, not sent by us:

if pending? -> true we never added the user to contacts -> we show it as pending
if pending is false -> we already added the user to contacts -> we show it as accepted

or I am missing something?

Reply:
ok so with system-tag this can all be replaced by `pending?`